core/frontend-devtools/src/tools/FrustumDecoration.ts

```diff
@@ -76,7 +76,7 @@ class FrustumDecoration {
 
   public static drawPreloadFrustum(builder: GraphicBuilder, frustum: Frustum) {
     const preloadColor = ColorDef.create(ColorByName.coral);
-    builder.setSymbology(preloadColor, preloadColor, 1, 2);
+    builder.setSymbology(preloadColor, preloadColor, 1, 2 as LinePixels);
```

Comment:
Should this really be 2 and not LinePixels.Code2 or something? @pmconne

Reply:
```suggestion
    builder.setSymbology(preloadColor, preloadColor, 1, LinePixels.Code2);
```